var/spack/repos/builtin/packages/acfl/package.py

```diff
@@ -131,6 +131,8 @@ class Acfl(Package):
     # Build Versions: establish OS for URL
     acfl_os = get_os()
 
+    test_requires_compiler = True
+
```

Comment:
This property is only used for post-installation tests that are run days to weeks after the installation (see https://spack.readthedocs.io/en/latest/packaging_guide.html#enabling-test-compilation).  

What you are adding here is an always on, build-time check (see https://spack.readthedocs.io/en/latest/packaging_guide.html#adding-installation-phase-tests).

Reply:
This was left here by mistake from when I was experimenting. I am going to remove this line.

Reply:
Ping me when this is removed.

Reply:
Done.